config/ax_flux_core.m4

```diff
@@ -83,9 +85,10 @@ AC_DEFUN([AX_FLUX_CORE], [
   ]
 
   #  Check for other flux-core libraries
-  #PKG_CHECK_MODULES([FLUX_IDSET], [flux-idset], [], [])
-  #PKG_CHECK_MODULES([FLUX_SCHEDUTIL], [flux-schedutil], [], [])
-  #PKG_CHECK_MODULES([FLUX_OPTPARSE], [flux-optparse], [], [])
+  PKG_CHECK_MODULES([FLUX_IDSET], [flux-idset], [], [])
+  PKG_CHECK_MODULES([FLUX_SCHEDUTIL], [flux-schedutil], [], [])
+  PKG_CHECK_MODULES([FLUX_OPTPARSE], [flux-optparse], [], [])
```

Comment:
Probably don't need to check for libflux-schedutil.so and libflux-optparse.so.

Reply:
Thanks! I'll go ahead and remove those.